src/demo/ui.jsx

```diff
@@ -1505,6 +1504,11 @@ let Guide = class Guide extends React.Component {
                         : currentGuide.text}
                     </div>
                   </div>
+                  {currentGuide.style === 'Info' && (
+                    <Button style={styles.infoGuideButton} onClick={() => {}}>
```

Comment:
Does this button still work with an empty `onClick` handler?

Reply:
yep -- since any click (after typing has finished) closes the dialog, we got this for free 😂